components/eamxx/src/share/io/scorpio_input.hpp

```diff
@@ -1,10 +1,8 @@
 #ifndef SCREAM_SCORPIO_INPUT_HPP
 #define SCREAM_SCORPIO_INPUT_HPP
 
-#include "share/io/scream_scorpio_interface.hpp"
```

Comment:
This header does not need scorpio interfaces. Removing this caused compiler errors, so I had to add it in a few cpp files.